libs/interpreter-lib/src/validation-checks/runtime-parameter-literal.ts

```diff
@@ -83,18 +84,10 @@ function checkRuntimeParameterValueParsing(
       runtimeParameterName,
       valueType,
     );
-  if (runtimeParameterValue === undefined) {
-    const rawValue =
-      props.evaluationContext.runtimeParameterProvider.getRawValue(
-        runtimeParameterName,
-      );
-    props.validationContext.accept(
-      'error',
-      `Unable to parse the value "${
-        rawValue ?? ''
-      }" as ${valueType.getName()}.`,
-      { node: runtimeParameter },
-    );
```

Comment:
Why is this code removed? Did we introduce changes to the runtime parameter handing with this PR?